src/commands/register.ts

```diff
@@ -26,10 +27,14 @@ export default class ComponentRegister extends Command {
 
   static flags = {
     ...Command.flags,
+    arg: flags.string({
+      char: 'a',
```

Comment:
They're very different commands but I feel like it might be confusing that we also use `-a` for the account flag in so many other places

Reply:
true - I actually meant to not include the shorthand. good call